profile/README.md

```diff
@@ -0,0 +1,38 @@
+# Welcome to the Content Queue!
+
+![Banner](./images/banner.webp)
+
+The Content Queue allows you to manage your social media posts by curating content in a GitHub workflow. Everything is managed through a GitHub Project board and GitHub Actions.
+
+## Common Workflow
+
+### Setup
+
+Check the [template repository](https://github.com/content-queue/template) for ready available workflows and instructions on how to set them up. Add these to a new repository and go through the credentials setup instructions. Set up the project board with the necessary columns as configured in the GitHub Actions workflows. And don't forget to give access to everyone who needs access to the repo.
```

Comment:
Instructions will be done next :)

Reply:
Do you want to adjust this section for the template repository thing?

Reply:
Currently working on the `README` for the template repo which will have all of that info, but we could be a bit more clear on this. Maybe we could even remove the following completely as that would be part of the instructions in the template repo anyway?

> Add these to a new repository and go through the credentials setup instructions. Set up the project board with the necessary columns as configured in the GitHub Actions workflows.

Reply:
Adjusted.